pkg/cli/admin/release/extract_tools.go

```diff
@@ -637,6 +644,7 @@ func (o *ExtractOptions) extractCommand(command string) error {
 	releaseArch := release.Config.Architecture
 	releaseName := release.PreferredName()
 	refExact := release.ImageRef
+	releaseArchitecture := release.ReleaseArchitecture()
```

Comment:
Is it possible to use this function https://github.com/openshift/oc/blob/0bea0594b2cdb44de33be3a96f8ac2cf68e10ee2/pkg/cli/admin/release/info.go#L749-L767 instead of defining a new one that is subset of the `Platform`?